packages/remix/src/utils/serverAdapters/express.ts

```diff
@@ -40,6 +42,10 @@ function wrapExpressRequestHandler(
     res: ExpressResponse,
     next: ExpressNextFunction,
   ): Promise<void> {
+    if (!pkg) {
+      pkg = await import(`${cwd()}/node_modules/react-router-dom`);
```

Comment:
Will is break if folks are bundling stuff (like in a serverless function)?

Can we just call import directly on `react-router-dom`?

Reply:
> Will is break if folks are bundling stuff (like in a serverless function)?

I think it depends on the packager used, but it seems there are some workarounds if such thing happens: https://www.serverless.com/plugins/serverless-jetpack#handling-dynamic-import-misses. It also applies to `loadModule` (using module.require) as far as I understand.

> Can we just call import directly on react-router-dom?

Tried that and it's working, but while building `@sentry/remix` I got a warning from Rollup because, it actually tried to resolve the deps from `sentry-javascript/node_modules` on build time.